AggregateMetrics/AggregateMetrics/AzureWebApp/Implementation/CounterFactory.cs

```diff
@@ -9,42 +9,57 @@
     internal class CounterFactory
     {
         /// <summary>
+        /// Set metrics alias to be the value given by the user.
+        /// </summary>
+        /// <param name="counterName">Name of the counter to retrieve.</param>
+        /// <param name="reportAs">Alias to report the counter.</param>
+        /// <returns>Alias that will be used for the counter.</returns>
+        private string SetCounterName(string counterName, string reportAs)
```

Comment:
this function returns something. Call it `GetCounterReportAsName`. Also can you pls copy logic from here: https://github.com/Microsoft/ApplicationInsights-dotnet-server/blob/master/Src/PerformanceCollector/Shared/PerformanceCollectorModule.cs#L491 to align how the counters will be named.